### PR TITLE
SCRUM-1709 Don't clear form on clicking 'Save & Add Another'

### PR DESCRIPTION
**Pull request by markquintontulloch**

### CODE REVIEW

File: src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useImmerReducer } from "use-immer";
 
 const initialNewAnnotationState = {
-	newAnnotation: {},
+	newAnnotation: {"negated": false},
```
Comment (oblodgett) [inline src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: Does this field need to be specified here?

Comment (markquintontulloch) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: I think so, otherwise it gets cleared when you do CLEAR

Comment (oblodgett) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: We were trying to not set any fields in the object, and just handle any issues elsewhere.

Comment (oblodgett) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: You could just default it in the clear call.

Comment (markquintontulloch) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: I tried just setting it in the 'CLEAR' case (as it was previously done in the 'OPEN_DIALOG' case) but it was throwing errors the second time you hit clear: 
`Uncaught TypeError: Cannot assign to read only property 'negated' of object '#<Object>'`

Comment (oblodgett) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: So doing a `draft.newAnnotation["negated"] = false;` in the CLEAR was throwing that error?

Comment (markquintontulloch) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: yep, but only on the 2nd time you hit Clear

Comment (markquintontulloch) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js:4]: and only if you'd entered something in one of the non-list fields

**Review by oblodgett — APPROVED** [review]

**Comment by markquintontulloch** [comment]
Redundant due to code incorporated in SCRUM-1903

**Pull request closed without merging**